crates/holochain/src/core/workflow/sys_validation_workflow/validation_deps.rs

```diff
@@ -80,23 +83,13 @@ impl ValidationDependencies {
         }
     }
 
-    pub fn get_mut(&mut self, hash: &ActionHash) -> Option<&mut ValidationDependencyState> {
```

Comment:
This wasn't used anywhere, removed. This is why it's really important to scope the visibility of everything down to a minimum because Clippy won't pick up that public types aren't used within the project and report it... because they might be used outside the project. Except this type is completely internal to sys validation... Mess!